packages/blockchain-utils-validation/src/blockchains/near.ts

```diff
@@ -0,0 +1,26 @@
+import nacl from "tweetnacl";
```

Comment:
Please use `@stablelib/ed25519` instead of `tweetnacl`.

Reply:
good idea, thanks